packages/context-fetch/src/withFetchToken.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+import { compose } from '@axa-fr/react-oidc-core';
+import { useReactOidc } from '@axa-fr/react-oidc-context';
+import { fetchToken } from '@axa-fr/react-oidc-fetch-core';
+
+type WindowFetch = typeof fetch;
+
+const withUser = (WrappedComponent: React.ComponentType) => (props: any) => {
+  const { oidcUser } = useReactOidc();
+  return <WrappedComponent {...props} user={oidcUser} />;
+};
+
+const withTokenFromFetchToken = (fetch: WindowFetch) => (WrappedComponent: React.ComponentType) => (props: any) => {
+  const { fetch: newFetch } = fetchToken(fetch)(props);
```

Comment:
**suggestion:** `newFetch` could be named something like `fetchWithToken`

Reply:
Good point